datastore/kubernetes.go

```diff
@@ -282,6 +282,38 @@ func (s *DataStore) UpdateLease(lease *coordinationv1.Lease) (*coordinationv1.Le
 	return s.kubeClient.CoordinationV1().Leases(s.namespace).Update(context.TODO(), lease, metav1.UpdateOptions{})
 }
 
+func (s *DataStore) ClearDelinquentState(volumeName string) (err error) {
+	defer func() {
+		err = errors.Wrapf(err, "failed to check ClearDelinquentState")
```

Comment:
"failed to check ClearDelinquentState" -> "failed to ClearDelinquentState".

Reply:
Fixed. Thank you